src/Main.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import './Main.css';
+import NewTaskForm from './NewTaskForm';
+import AddedTasks from './AddedTasks';
+import EditTaskForm from './EditTaskForm';
+
+export default class Main extends Component {
+  state = {
+    isOpenNewTaskForm: false,
+    isOpenEditTaskForm: false,
+    tasks: [],
+    dataEditTask: null
+  };
+
+  render() {
+    const newTaskForm = this.state.isOpenNewTaskForm && <NewTaskForm 
+      showNewTaskForm={this.showNewTaskForm} 
+      getDataNewTask={this.getDataNewTask}
+      />;
+    const editTaskForm = this.state.isOpenEditTaskForm && <EditTaskForm 
+      showEditTaskForm={this.showEditTaskForm} 
+      dataEditTask={this.state.dataEditTask}
+      editTask={this.editTask}
+    />;
+    const addedTasks = <AddedTasks 
+      tasks={this.state.tasks}
+      getDataEditableTask={this.getDataEditableTask} 
+      showEditTaskForm={this.showEditTaskForm}
+      removeTask={this.removeTask}
+    />;
+
+    return (
+      <main>
+        <div className='added-tasks_container'>
+          <div className='added-tasks_container__header'>
+            <button className="add-new-task__button" onClick={this.showNewTaskForm}>Add Task</button>
+            <span className='added-tasks__span' onClick={this.stateStatus}>Added tasks:</span>
+          </div>
+          {newTaskForm}
+          {editTaskForm}
+          {addedTasks}
+        </div>
+      </main>
+    );
+  };
+
+  showNewTaskForm = () => {
+    this.setState({
+      isOpenNewTaskForm: !this.state.isOpenNewTaskForm
```

Comment:
You directly change state in ```!this.state.isOpenNewTaskForm``` expression. The idea of state changing, that you can't do it directly. You should use destructuring to avoid this situation:
```
const { isOpenNewTaskForm } = this.state;

this.setState({
    isOpenNewTaskForm: !isOpenNewTaskForm
});
```